.jenkins/Jenkinsfile

```
@@ -9,6 +9,7 @@ commitHash = params.commitHash ? params.commitHash : params.sha1
 PROFILE = params.profile.trim().toLowerCase()
 ARCHITECTURE = params.architecture.trim().toLowerCase()
 GITHUB_URL = "git@github.com:stratum/fabric-tna.git"
+SPECIFIC_PROFILE = "${PROFILE}-${ARCHITECTURE}"
```

For some reason, "specific profile" sounds confusing. Would it be more accurate to refer to this as a make target?
```suggestion
MAKE_TARGET = "${PROFILE}-${ARCHITECTURE}"
```